spec/services/charges/charge_models/volume_service_spec.rb

```diff
@@ -87,8 +135,16 @@
       aggregation_result.full_units_number = 300
     end
 
-    it 'applies unit amount the third range' do
+    it 'applies unit amount the third range', :aggregate_failures do
       expect(apply_volume_service.amount).to eq(149.3)
+      expect(apply_volume_service.unit_amount.round(2)).to eq(0.50)
+      expect(apply_volume_service.amount_details).to eq(
+        {
+          flat_amount: 50,
+          per_unit_amount: 0.331,
+          per_unit_total_amount: 99.3,
+        }
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)